### PR TITLE
DEV-6348: publish certify history submission attributes

### PR DESCRIPTION
**Pull request by alburde1**
**Description:**
Adding the history of publication/certification to the submission attributes table.

**Technical details:**
The technical details for the knowledge of other developers. Any detailed caveats or specific deployment steps should be outlined here.

**Requirements for PR merge:**

1. [x] Unit & integration tests updated
2. [x] API documentation updated (N/A)
3. [x] Necessary PR reviewers:
    - [x] Backend
4. [x] Matview impact assessment completed (N/A)
5. [x] Frontend impact assessment completed (N/A)
6. [x] Data validation completed
7. [x] Appropriate Operations ticket(s) created
8. [x] Jira Ticket [DEV-6348](https://federal-spending-transparency.atlassian.net/browse/DEV-6348):
    - [x] Link to this Pull-Request
    - [x] Performance evaluation of affected (API | Script | Download)
    - [x] Before / After data comparison (N/A)

**Area for explaining above N/A when needed:**
```
```

### CODE REVIEW

File: usaspending_api/etl/tests/integration/test_load_multiple_submissions.py [hunk 1]
```diff
@@ -159,8 +159,8 @@ def setUp(self):
                     submission_id,
                     updated_at
                 ) (values
-                    (1, 1, '2000-01-01'), (2, 2, '2000-01-02'), (3, 3, '2000-01-03'), (4, 4, '2000-01-04'),
```
Comment (emily-brents) [inline usaspending_api/etl/tests/integration/test_load_multiple_submissions.py:162]: Is there a reason that this date had to be changed?

Comment (alburde1) [reply usaspending_api/etl/tests/integration/test_load_multiple_submissions.py:162]: I added another one with the same original date, it was just later in the list because I wanted to keep the "earlier" one a lower number. The one with an ID of 8 is now what this one was. I needed an entry with more than one publication to test things like null certify and multiple publications.

**Review by emily-brents — DISMISSED** [review]
LGTM

File: usaspending_api/etl/management/commands/load_submission.py [hunk 2]
```diff
@@ -157,6 +157,29 @@ def signal_handler(signal, frame):
     def get_broker_submission(self):
         self.db_cursor.execute(
             f"""
+                with publish_certify_history as (
+                    select
+                        distinct_pairings.submission_id,
+                        json_agg(
+                            json_build_object(
```
Comment (tony-sappe) [inline usaspending_api/etl/management/commands/load_submission.py:164]: I think `jsonb` is preferable.

`json_agg` -> `jsonb_agg`
`json_build_object` -> `jsonb_build_object`

According to the docs, the postgres field `JSONField` is actually a jsonb datatype: https://docs.djangoproject.com/en/2.2/ref/contrib/postgres/fields/#django.contrib.postgres.fields.JSONField

I assume the datatype conversion to USAspending table storage will be seamless but it is probably better to be consistent.

File: usaspending_api/etl/management/commands/load_submission.py [hunk 3]
```diff
@@ -157,6 +157,29 @@ def signal_handler(signal, frame):
...
+                                'published_date', ph.updated_at::timestamptz,
+                                'certified_date', ch.updated_at::timestamptz
```
Comment (tony-sappe) [inline usaspending_api/etl/management/commands/load_submission.py:166]: 👍

**Review by emily-brents — APPROVED** [review]
Still looks good to me
